### DISCUSSION TITLE
i need to create fields base on click + icone

### DISCUSSION THREAD

**Discussion by tetswww** [Budibase/budibase#12264] [Help] [answered]
**Describe the feature request**
i need to create fields base on click + icone 
like 
![image](https://github.com/Budibase/budibase/assets/149157189/4433d454-5da9-4dbb-94a9-c7ad27381fed)
 this image 

https://screenpal.com/watch/c0XeiJVERTo

i need create fields like this video 

give suggestions

**Comment by ConorWebb96** [comment, marked as answer]
Hey @tetswww,

There is an open issue already for dynamic field components this can be found [here](https://github.com/Budibase/budibase/issues/5658). You won't be able to do this until its added to the core product.
